spec/dummy_app/db/migrate/20110208155312_set_up_test_tables.rb

```diff
@@ -61,6 +61,15 @@ def up
       t.timestamps null: true
     end
 
+    if ENV["DB"] == "postgres"
+      create_table :postgres_users, force: true do |t|
+        t.string     :name
+        t.integer    :post_ids,    array: true
+        t.datetime   :login_times, array: true
```

Comment:
I like how you are testing two different types of arrays here. Also, it looks like this is the first test table to have arrays at all! Kind of surprising.